app/src/main/res/values/strings.xml

```diff
@@ -35,6 +35,7 @@
     <string name="settingsDefaultBrowserEnabled">Set as Default Browser</string>
     <string name="defaultBrowserMaybeLater">Maybe Later</string>
     <string name="defaultBrowserLetsDoIt">Set as Default Browser</string>
+    <string name="defaultBrowserMenuTitle">Set as Default Browser</string>
```

Comment:
I assumed that we want to keep each text view's string separate to avoid potential changes impacting other text views unexpectedly. And since we already had the same string translated, I copied that around.

Reply:
I added a commit that consolidates the strings instead.